multicluster/charts/linkerd-multicluster/values.yaml

```diff
@@ -154,3 +157,52 @@ localServiceMirror:
 
   # -- Resources for the Service mirror container
   resources: {}
+
+# -- List of service mirror controllers.
+# References to the Links deployed in the cluster, each of which will have a
+# corresponding service mirror controller deployed.
+# Only `link.ref.name` is required for each entry.
+# Example (all the missing values take their values from controllerDefaults):
+# controllers:
+# - link:
```

Comment:
can we add some fields to this example to show how to set values which override the defaults?

Reply:
Yep, will do :+1: